.github/workflows/nancy.yml

```diff
@@ -0,0 +1,12 @@
+name: Nancy
```

Comment:
I propose you use a name that describes what the tool does instead of the name of the tool. I feel like the specific tool used is an "implementation detail".